google/pubsub/v1/pubsub_gapic.yaml

```diff
@@ -6,7 +6,7 @@ language_settings:
   python:
     package_name: google.pubsub.v1
   go:
-    package_name: google.golang.org/cloud/pubsub
+    package_name: cloud.google.com/go/pubsub/apiv1
```

Comment:
Just checking: should this really be "apiv1" or should that last path element be split up or deleted altogether? 

FWIW, https://cloud.google.com/go/pubsub redirects to https://godoc.org/cloud.google.com/go/pubsub, and Jonathan's email suggests simply replacing s|google.golang.org/cloud/|cloud.google.com/go/| in import paths.